models/ios/ntp_global/ios_ntp_global.yaml

```
@@ -0,0 +1,229 @@
+            description:  Allow only control queries
+            type: dict
+            suboptions: 
+              access_list: *access_group_acl
+              kod: *kod
+          serve: 
+            description:  Provide server and query access
+            type: dict
+            suboptions:
+              access_list: *access_group_acl
+              kod: *kod
+          serve_only: 
+            description:  Provide only server access
+            type: dict
+            suboptions:
+              access_list: *access_group_acl
+              kod: *kod
+      allow:
+        description: Allow processing of packets
+        type: dict
+        elements: dict
+        suboptions:
+          control:
+            description: Allow processing control mode packets
+            type: bool
+          private:
+            description: Allow processing private mode packets
+            type: bool
+          rate_limit_delay:
+            description: Rate limiting delays
+            type: int
+      authenticate:
+        description: Authenticate time sources
+        type: bool
+      authentication_keys:
+        description: Authentication key for trusted time sources
+        type: list
+        elements: dict
+        suboptions:
+          id:
+            description: Key number
+            type: int
+          authentication_type:
+            description: Authentication type
+            type: str
+          key:
+            description: Password
+            type: str
+            no_log: True
+          encryption_type:
+            description: Authentication key encryption type
+            type: int
+      broadcast_delay:
+        description: Estimated round-trip delay
+        type: int
+      clock_period:
+        description: Length of hardware clock tick, clock period in 2^-32 seconds
+        type: int
+      logging:
+        description: Enable NTP message logging
+        type: bool
+      master:
+        description: Act as NTP master clock
+        type: dict
+        elements: dict
+        suboptions:
+          enabled:
+            description: Enable master clock
+            type: bool
+          stratum_number:
+            description: Stratum number
+            type: int
+      max_associations:
+        description: Set maximum number of associations
+        type: int
+      max_distance:
+        description: Maximum Distance for synchronization
+        type: int
+      min_distance:
+        description: Minimum distance to consider for clockhop
+        type: int
+      orphan:
+        description: Threshold Stratum for orphan mode
+        type: int
+      panic_update:
+        description: Reject time updates > panic threshold (default 1000Sec)
+        type: bool
+      passive:
+        description: NTP passive mode
+        type: bool
+      peers:
+        description: Configure NTP peer
+        type: list
+        elements: dict
+        suboptions: &peer
+          host:
```

For consistency with how we have historically dealt with these attributes (neighbors -> neighbor), can we call it peer?
```suggestion
          peer:
```